modules/api-custom-image-creating.adoc

```diff
@@ -77,8 +79,10 @@ The example YAML file includes the following information:
 <6> The version for the image. You can configure multiple versions for the same image. For this example, the version is 1.0.
 <7> An annotation that gives the user information about the Python packages and versions that are pre-installed in the image.
 <8> An annotation that specifies information such as the Python version, Jupyter version, or CUDA version.
-<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`.
-<10> The image registry URL to store the image stream. Store the image stream in an image registry outside of the cluster where {productname-short} is installed.
+<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`. 
+<10> An annotation that specifies whether the image version has tags that are outdated and out of the regular maintenance cycle.
+<11> An annotation that is a reference to the build commit with the ID for the updated information.
```

Comment:
I personally consider

> commit with the ID for the updated information

to be highly confusing statement, but I'm not up to coming up with a better formulation on the spot.

It's what will be shown in parens in the Dashboard like this

![image](https://github.com/user-attachments/assets/9298a200-4ab8-44c5-a6c7-6ea947803153)

Reply:
Not necessarily the latest content. It should point to the sources from which that particular tag was built.